code/scripting/lua/LuaTable.h

```diff
@@ -19,7 +19,7 @@ class LuaTable;
  * be in the exact same state when you call the next method of this class.
  */
 class LuaTableIterator {
-	lua_State* _L = nullptr;
+	lua_State* L_State = nullptr;
```

Comment:
Only global names beginning with an underscore are reserved. For names in non-global namespaces (e.g. a class) the name may begin with an underscore but the next letter must be lower case which also explains why `_L` is an invalid name.